### PR TITLE
refactor: Rename FixedPriceOracle to FixedUintValue

### PR DESCRIPTION
**Pull request by davidprepo**
See spec: https://www.notion.so/prepo/SC-Backlog-Tasks-0c6725bb870f4c22b7ba771347333662#796ba14e05964b5cacf9fced644709f7

Looks like the refactoring inside the files has caused github to decide this is a delete/add operation on the files rather than a rename, unfortunately.

### CODE REVIEW

File: packages/prepo-shared-contracts/contracts/FixedUintValue.sol [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pragma solidity =0.8.7;
+
+import "./interfaces/IFixedUintValue.sol";
+import "./SafeOwnable.sol";
+
+contract FixedUintValue is IFixedUintValue, SafeOwnable {
+  uint256 private _value;
+
+  function setValue(uint256 newValue) external override onlyOwner {
+    _value = newValue;
```
Comment (XavierEkkel) [inline packages/prepo-shared-contracts/contracts/FixedUintValue.sol:11]:
```suggestion
  function set(uint256 value) external override onlyOwner {
    _value = value;
```

File: packages/prepo-shared-contracts/test/FixedUintValue.test.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    it('sets owner to deployer', async () => {
+      expect(await fixedUintValue.owner()).to.eq(deployer.address)
+    })
+
+    it('sets nominee to zero address', async () => {
+      expect(await fixedUintValue.getNominee()).to.eq(ZERO_ADDRESS)
+    })
+  })
+
+  describe('# setValue', () => {
```
Comment (XavierEkkel) [inline packages/prepo-shared-contracts/test/FixedUintValue.test.ts:35]:
```suggestion
  describe('# set', () => {
```

**Review by XavierEkkel — APPROVED** [review]
LGTM after comments, can merge after comments

File: packages/prepo-shared-contracts/test/fixtures/FixedUintValueFixture.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { ethers } from 'hardhat'
+import { FixedUintValue } from '../../types/generated'
+
+export async function fixedUintValueFixture(): Promise<FixedUintValue> {
+  const Factory = await ethers.getContractFactory('FixedUintValue')
+  return (await Factory.deploy()) as FixedUintValue
```
Comment (ramenforbreakfast) [inline packages/prepo-shared-contracts/test/fixtures/FixedUintValueFixture.ts:6]:
```suggestion
  const factory = await ethers.getContractFactory('FixedUintValue')
  return (await factory.deploy()) as FixedUintValue
```
I know else where this has been merged this way, but been meaning to switch them over to just camelcase, or lowercase in this case.

**Review by ramenforbreakfast — APPROVED** [review]
LGTM besides nits.

**Pull request merged**